kili/services/label_import/importer/__init__.py

```diff
@@ -0,0 +1,243 @@
+
+        labels = []
+        if labels_files is not None and len(labels_files) > 0:
+            label_paths = get_file_paths_to_upload(
+                [str(f) for f in labels_files],
+                lambda path: path.endswith(self._get_label_file_extension()),
+                verbose=self.logger_params.level in ["INFO", "DEBUG"],
+            )
+            if len(label_paths) == 0:
+                raise ValueError(
+                    "No label files to upload. "
+                    "Check that the paths exist and file types are .json"
+                )
+            external_ids = [get_external_id_from_file_path(Path(path)) for path in label_paths]
+            labels = [
+                LabelToImport(path=p, label_asset_external_id=e_id)
+                for (p, e_id) in zip(label_paths, external_ids)
+            ]
+
+        elif labels_file_path is not None:
+            labels = self._read_labels_file_path(labels_file_path)
+
+            if len(labels) == 0:
+                raise ValueError(f"No label file was found in csv: {labels_file_path}")
+        return labels
+
+    @classmethod
+    def _read_labels_file_path(cls, labels_file: Path) -> List[LabelToImport]:
+        """
+        Read CSV and fill a list of labels to import.
+        """
+        data = []
+        with labels_file.open("r") as l_f:
+            csv_reader = csv.reader(l_f)
+            data = []
+            headers = []
+            for ind, row in enumerate(csv_reader):
+                if ind == 0:
+                    headers = row
+                else:
+                    headers_row = zip(headers, row)
+                    str_dict = dict(headers_row)
+
+                    typed_dict: Dict[str, Any] = {
+                        "path": str_dict["path"],
+                    }
+                    if len(str_dict.get("label_asset_id", "")):
+                        typed_dict["label_asset_id"] = str_dict["label_asset_id"]
+
+                    if len(str_dict.get("label_asset_external_id", "")):
+                        typed_dict["label_asset_external_id"] = str_dict["label_asset_external_id"]
+
+                    if len(str_dict.get("label_type", "")):
+                        typed_dict["label_type"] = cast(LabelType, str_dict["label_type"])
+
+                    if len(str_dict.get("seconds_to_label", "")):
+                        typed_dict["seconds_to_label"] = int(str_dict["seconds_to_label"])
+
+                    if len(str_dict.get("author_id", "")):
+                        typed_dict["author_id"] = str_dict["author_id"]
+
+                    data.append(LabelToImport(**typed_dict))
+        return data
+
+
+class YoloLabelImporter(AbstractLabelImporter):
+    """
+    Label importer in the yolo format.
+    """
+
+    def _get_label_file_extension(self) -> str:
+        return ".txt"
+
+    @classmethod
+    def _read_classes_from_meta_file(
+        cls, meta_file_path: Optional[Path], input_format: LabelFormat
```

Comment:
why using a classmethod here if you don't use the cls ?

Reply:
oops forgot to change that